data/faq/application_template.md

```diff
@@ -0,0 +1,122 @@
+coala 2017 Application Template
+===============================
+
+```
+Student Info
+------------
+
+- Name:
+- GitHub username:
+- Alternative-/Nickname:
+- Email:
+- Telephone(mobile or home):
```

Comment:
I don't think google collects phone. FWIW I'd love to collect phone for mentors definitely. I know it's kind of a privacy thing but there's an actual reason for this:

we, mainly @sims1253 , will have to kick them to fill the evaluations. Take a large enough mentor fleet and you will find some mentor who isn't available during the evaluations at all via mail, IRC and Gitter. For that reason we're collecting phone numbers for mentors at GNOME and PSF so we can reach them in an emergency.

I don't see this happening to the student because when it happens then his failure due to him not being reachable will be his fault, not ours.